lib/stages/pulsarPostProcess.cpp

```diff
@@ -73,27 +73,32 @@ pulsarPostProcess::~pulsarPostProcess() {
 
 void pulsarPostProcess::fill_headers(unsigned char* out_buf, struct VDIFHeader* vdif_header,
                                      const uint64_t fpga_seq_num, struct timespec* time_now,
-                                     struct psrCoord* psr_coord, uint16_t* freq_ids) {
+                                     struct psrCoord* psr_coord, uint16_t* thread_ids) {
+
+    // Get the Telescope instance and pre-calc the length of an FPGA frame
+    auto& tel = Telescope::instance();
+    uint64_t fpga_ns = tel.seq_length_nsec();
+    float fpga_s = 1e-9 * fpga_ns;
```

Comment:
const for both `fpga_ns` and `fpga_s`?

Reply:
actually, isn't fpga_s == `tel.seq_length()`?

Reply:
It returns a timespec though. So I'd need to run it through ts_to_double.